### PR TITLE
Article Page Working Branch

### PR DESCRIPTION
**Pull request by Joseph-Burke**
**Pull request to merge branch "Article page #1" with the master branch.**

"Article page #1" branch contains our entire history of changes made to the project since initialising this GitHub project. 

This Pull Request will update the project with all of its content and styling: 

- header
- main
- text
- footer
- images
- links
- videos
- everything!

The authors are requesting that these changes be merged into the master branch as soon as it meets all of the project's requirements.

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,694 @@
...
+#video-description,
+#video-credits {
+  font-size: var(--medium-font-size);
+  color: #666;
+  margin-top: 0.7rem;
+  font-weight: normal;
+  width: var(--default-width);
+}
+
+#video-credits {
```
Comment (andresporras3423) [inline css/style.css]: Don't overuse ID selectors

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,694 @@
...
+  margin-left: 50px;
+}
+
+.footer-right-section h5 {
+  margin-bottom: 15px;
+}
+
+.links-with-icon {
+  margin-bottom: 15px;
+}
```
Comment (andresporras3423) [inline css/style.css:647]: Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @Joseph-Burke @Amita-Roy** 
Awesome work in your first submission for this project :muscle:  
Your project looks really like a clone of the original :clap: :clap: :clap:
It meets most of the requirements. There are just a couple of details to improve before you can go ahead:
- [x] Don't overuse ID Selectors in your CSS
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).

#### OPTIONAL

- [x] it would be good if you make sure that the social icons, at the end of the article, can stay in the same line for large screen size (1200px or bigger). In your page, the social icons look like this:






Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras
(for any question you can contact me in slack as @oscarporras02)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1148 @@
...
+            ></path>
+          </svg>
+        </div>
```
Comment (peterrobert) [inline index.html]: You need to make the logo a link

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :mechanic: 
**HELLO** @Joseph-Burke and @Amita-Roy 

Very impressive work you have met all the requirements for this project except one. Before I can approve this project kindly make that one change and resubmit your work

If you have any questions regarding the review kindly reach me via slack on @peterrobert

### HAPPY CODING :100:

File: css/style.css (hunk 1 above)
Comment (Joseph-Burke) [reply css/style.css]: All former ID selectors are now class selectors.

File: css/style.css (hunk 2 above)
Comment (Joseph-Burke) [reply css/style.css:647]: All possible grouping of selectors has been done, within what is permitted by the CSS linter.

File: index.html (hunk 3 above)
Comment (Joseph-Burke) [reply index.html]: We have now wrapped the <svg> element in an <a> element that links to the New York Times main/landing page.

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle: 

### Hello @Joseph-Burke and @Amita-Roy    :hand: 

### Good job your project is getting better and better. :+1: 

### Unfortunately, there are still one change that must be done to get the project approved:
- [x] Pull Request has meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- You have to keep a professional github repository in all your projects:
Screenshot from 2020-06-17 10-08-36
Please provide a shortly description about what this PR introduce.

### once done with the changes, please ask for another code review.

**Review by peterrobert — APPROVED** [review]
### APPROVED :100: :100: 
Great project. You have met all the required changes. 

### [OPTIONAL]
 
Before merging the project into master. Kindly, add bullet points in your pull request message. Explain what you have worked on as bullet points. Its much better that way.

**HAPPY CODING** :100:

**Pull request merged**